modules/cloudfront.js

```diff
@@ -22,11 +22,21 @@ const CLOUDFRONT_PATHS = [
     { path: '/user/*', isPostable: true, queryStrings: ['redirectUrl', 's', 'token'] }
 ];
 
+if (config.get('features.enableLegacyFileArchiving')) {
```

Comment:
Not 100% sure this flag makes sense here. This will always be `true` as this is enabled in development and the cloudfront configs are only ever generated in development. Not sure what to suggest as an alternative approach though.